src/ui.rs

```diff
@@ -1224,6 +1225,13 @@ impl PofToolsGui {
                                         dock_name
                                     )
                                 }
+                                Warning::Detail0NonZeroOffset => {
+                                    if let Some(id) = self.model.header.detail_levels.get(0) {
+                                        format!("⚠ Detail0 object '{}' should have a (0, 0, 0) offset.", self.model.sub_objects[*id].name)
+                                    } else {
+                                        unreachable!()
+                                    }
```

Comment:
```suggestion
                                    let id = self.model.header.detail_levels[0];
                                    format!("⚠ Detail0 object '{}' should have a (0, 0, 0) offset.", self.model.sub_objects[*id].name)
```
if you are going to panic in the else case, you may as well use the panicking index operator